app/models/host_status/execution_status.rb

```diff
@@ -1,17 +1,21 @@
 class HostStatus::ExecutionStatus < HostStatus::Status
+  # execution finished successfully
   OK = 0
```

Comment:
What's the advantage of numbers over symbols. Just minor readability enhancement. If nothing else, debugging the variable with the status would be nicer.